cloud/amazon/ec2_elb_lb.py

```diff
@@ -344,6 +351,13 @@
 except ImportError:
     HAS_BOTO = False
 
+try:
+    import boto3
+    import botocore
+    HAS_BOTO3 = True
+except ImportError:
+    HAS_BOTO3 = False
+
```

Comment:
The import statement should be nested with the rest of the tag code.  We don't want to break existing playbooks where people haven't got boto3 and they don't need the tag functionality.